explore/src/main/scala/explore/config/AdvancedConfigurationPanel.scala

```diff
@@ -358,48 +403,74 @@ sealed abstract class AdvancedConfigurationPanelBuilder[
                   .bigDecimal(integers = 3.refined, decimals = 1.refined)
                   .toSequence()
                   .optional,
-                units = "nm"
+                units = customizedUnit("nm", "Uknown", view.get.isDefined),
+                disabled = disableSimpleEdit,
+                clazz = ExploreStyles.WarningInput.when_(view.get.isDefined)
               ).clearable
             )
+          }
 
-          def offsetsControl(onChange: Callback): VdomElement =
+          def offsetsControl(onChange: Callback): VdomElement = {
+            val view = explicitSpatialOffsets(props.scienceModeAdvanced)
             ReactFragment(
-              <.label("Spatial Offsets", HelpIcon("configuration/spatial-offsets.md".refined)),
+              <.label("Spatial Offsets",
+                      HelpIcon("configuration/spatial-offsets.md".refined),
+                      ExploreStyles.SkipToNext
+              ),
               InputWithUnits(
                 id = "offsets".refined,
-                value = explicitSpatialOffsets(props.scienceModeAdvanced).withOnMod(_ => onChange),
+                value = view.withOnMod(_ => onChange),
                 validFormat = ExploreModelValidators.offsetQNELValidWedge,
                 changeAuditor = ChangeAuditor
                   .bigDecimal(integers = 3.refined, decimals = 2.refined)
                   .toSequence()
                   .optional,
-                units = "arcsec"
+                units = customizedUnit("arcsec", "Uknown", view.get.isDefined),
+                disabled = disableSimpleEdit,
+                clazz = ExploreStyles.WarningInput.when_(view.get.isDefined)
               ).clearable
             )
+          }
 
           val invalidateITC: Callback =
             props.potITC.set(Pot.pending[Option[ITCSuccess]])
 
           val zeroDuration: NonNegDuration = NonNegDuration.unsafeFrom(Duration.ofMillis(0))
 
+          val wavelengthView         = overrideWavelength(props.scienceModeAdvanced)
+          val originalWavelengthText =
+            props.spectroscopyRequirements.wavelength.fold("None")(w =>
+              f"${Wavelength.decimalMicrometers.reverseGet(w)}%.3f"
+            )
+
+          val originalSignalToNoiseText =
+            props.spectroscopyRequirements.signalToNoise.fold("None")(sn =>
+              s"S/N ${InputValidWedge.truncatedPosBigDecimal(0.refined).reverseGet(sn)}"
+            )
+
           def onModeMod(modType: Option[ExposureTimeModeType]): Callback = {
             val optITC: Option[ITCSuccess] = props.potITC.get.toOption.flatten
             val oetm                       = modType.map {
               case ExposureTimeModeType.SignalToNoise =>
                 val sn = signalToNoiseView
                   .map(_.get)
+                  .orElse(props.spectroscopyRequirements.signalToNoise)
                   .orElse(optITC.map(_.signalToNoise))
                   .getOrElse(PosBigDecimal.unsafeFrom(BigDecimal(100)))
```

Comment:
Can this be `BigDecimal(100).refined`?

Reply:
Yes, I guess that can work now with lucuma-refined. Although it needed a bit of type assistance.